packages/backend/src/workers/action.ts

```diff
@@ -145,28 +145,32 @@ worker.on('failed', async (job, err) => {
       job: job.data,
     },
   )
+  try {
+    const flow = await Flow.query()
+      .findById(job.data.flowId)
+      .withGraphFetched('user')
+      .throwIfNotFound()
+
+    const shouldAlwaysSendEmail =
```

Comment:
nit: probably can shorten to `alwaysSendEmail`

Reply:
i usually add a should/is for booleans for readability sia, is that bad?